lib/libc/Kconfig

```diff
@@ -47,7 +47,7 @@ config NEWLIB_LIBC_FLOAT_SCANF
 config STDOUT_CONSOLE
 	bool "Send stdout to console"
 	depends on CONSOLE_HAS_DRIVER
-	default NEWLIB_LIBC
+	default y
```

Comment:
Why this change specifically?

Reply:
having libc stdout functions do what is expected seems a reasonable default